examples/resnet_app.py

```diff
@@ -18,6 +25,9 @@
         conda install cudatoolkit=11.0 -y
         pip install tensorflow==2.4.0 pyyaml
         pip install protobuf==3.20
+        mkdir -p $CONDA_PREFIX/etc/conda/activate.d
+        echo 'CUDNN_PATH=$(dirname $(python -c "import nvidia.cudnn;print(nvidia.cudnn.__file__)"))' >> $CONDA_PREFIX/etc/conda/activate.d/env_vars.sh
+        echo 'export LD_LIBRARY_PATH=$CONDA_PREFIX/lib/:$CUDNN_PATH/lib:$LD_LIBRARY_PATH' >> $CONDA_PREFIX/etc/conda/activate.d/env_vars.sh
```

Comment:
Nice! Thanks for identifying the issue to be failing to find the path to `cudnn`. Could we try to install the cudnn in Line 25, i.e., `conda install cudatoolkit=11.0 cudnn=8.1.0 -y`, instead of manually adding the those paths, as asking the user to manually do that seems a bit too demanding?